src/InteractiveView.js

```diff
@@ -0,0 +1,14 @@
+import View from './View';
+
+const INTERACTIVE_NAVIGATION_START_MARK_NAME = 'transitionStart';
+
+export default class InteractiveView extends View {
+	startMarkName = null;
+
+	startTransition() {
```

Comment:
I'm curious about the decision to give 'transition' responsibilities to a particular view rather than implementing it exclusively in the `UXCapture` 'controller' API.

Currently, the `View` class is only responsible for organizing its `Zone`s, which makes its role relatively easy to reason about, and the `UXCapture` interface is responsible for managing 'state updates' related to manipulating views. `startTransition` seems much more aligned with the role of the controller interface from that perspective. Similarly, the 'state' of the `startMarkName` doesn't really seem like something that a `View` should have to manage - the view instance actually never refers to that value directly, so it's sort of a surprising place to keep it, and the `UXCapture` interface is already statefully aware of the current view, so that also seems like a good place to keep track of the current `startMarkName`, given that it's a 'global' state that could affect the whole system - each view instance doesn't need a different `startMarkName`, so no need to store it as an instance property.

Essentially, I recommend that `UXCapture.startTransition` be implemented as:

```js
// module-scoped
let _startMarkName = 'navigationStart';

startTransition: () => {
  UXCapture._clearMarksAndMeasures();  
  _startMarkName = 'transitionStart';
  window.performance.mark(_startMarkName);
}
```

Then, you don't need any `View` subclasses or `if (_view) {` logic.

in order to get the `_startMarkName` into the `Zone.measure` call, you can pass down `startMarkName` as a `View` prop that then gets passed to its zones:

```js
startView: zoneConfigs => {
  const viewProps = {
    startMarkName: _startMarkName,
    ...
  }
}
```